Features/icons change, rotem inclusion and some other final changes.

greggmiller opened the pull request:
![Simulator Screen Shot - iPhone 11 - 2020-06-11 at 11 47 57](https://user-images.githubusercontent.com/54889183/84336646-1236f300-abdb-11ea-8245-18226a1681e3.png)
![Simulator Screen Shot - iPhone 11 - 2020-06-11 at 11 47 47](https://user-images.githubusercontent.com/54889183/84336648-12cf8980-abdb-11ea-8ff5-25f32f9fb62d.png)
![Simulator Screen Shot - iPhone 11 - 2020-06-11 at 11 47 38](https://user-images.githubusercontent.com/54889183/84336649-13682000-abdb-11ea-9244-1682a62dc5a0.png)
![Simulator Screen Shot - iPhone 11 - 2020-06-11 at 11 47 02](https://user-images.githubusercontent.com/54889183/84336650-13682000-abdb-11ea-929c-a813bb071992.png)



All icons change to FontAwesome from Emojis in alignment with management request. Appearance improves professionalism.
Minor typos addressed.
Movement of your welfare section to Quick ICU training to reflect less significance.
Functional clinical support tool with high utility in patient management to help guide blood product administration. reduces information-dense algorithms into a simple calculator. Improves app functionality and use.
Infographic
![Simulator Screen Shot - iPhone 11 - 2020-06-11 at 11 48 47](https://user-images.githubusercontent.com/54889183/84336637-0d723f00-abdb-11ea-8240-b89440165fa6.png)
![Simulator Screen Shot - iPhone 11 - 2020-06-11 at 11 48 40](https://user-images.githubusercontent.com/54889183/84336642-106d2f80-abdb-11ea-8c9d-754206c43f09.png)
![Simulator Screen Shot - iPhone 11 - 2020-06-11 at 11 48 07](https://user-images.githubusercontent.com/54889183/84336644-119e5c80-abdb-11ea-919b-da1bf1ee18cf.png)


 page algorithms are derived from are also included.

 GitHub issues #343, #344, #346, #348, #349, #351, #352 all addressed.

These should be all major and minor changes required before a final beta release 👍 
There can be some improvements both in terms of refactoring, design and other aspects so feel free to comment. Images included.

☝️ Provide a short & descriptive title for this PR.

Provide a more detailed summary of the changes in this PR here.  What does it do?

- [ ] `release-notes.txt` has been updated.
(👆 delete this line if your changes have no user facing features)

## Changes

- Dot points explaining what is changed
- Did the thing
- Changed some stuff

## Screenshots

If your PR changes anything visual please paste in some screenshots of the app here.  Make sure you are working off the designs in zeplin.  If your PR makes some significant visual tweaks feel free to include one of the designers as a reviewer, so they can check off and comment on what is changing.

## Things to note

Include anything else the other Devs or Designers on the project should know?  Will there be work you are deferring now, which done in later follow up PRs?   Does this change anything about the code base they need to be aware of?

greggmiller committed:
Change of Icons, ongoing process.

greggmiller committed:
All icons change to FontAwesome.
GitHub issues #349, #348, #344, 343 all addressed.
Some ongoing other work still required

greggmiller committed:
All icons change to FontAwesome.
                                      GitHub issues #349, #348, #344, 343 all addressed.
                                      Some ongoing other work still required.

                                      Movement of Welfare and institution of ROTEm calculater.

greggmiller committed:
All icons change to FontAwesome.
                                      GitHub issues #349, #348, #344, 343 all addressed.
                                      Some ongoing other work still required.

                                      Movement of Welfare section.
                                      Insertion of ROTEM calculator adds functionality.
                                      Still Some minor tidy up work required.
                                      Will be completed tomorrow.

greggmiller committed:
All icons change to FontAwesome.
                                      GitHub issues #349, #348, #344, 343 all addressed.
                                      Some ongoing other work still required.

                                      Movement of Welfare section.
                                      Insertion of ROTEM calculator adds functionality.
                                      Still Some minor tidy up work required.
                                      Will be completed tomorrow.

greggmiller committed:
All icons change to FontAwesome.
                                      GitHub issues #349, #348, #344, 343 all addressed.
                                      Some ongoing other work still required.

                                      Movement of Welfare section.
                                      Insertion of ROTEM calculator adds functionality as clinical decision support tool.
                                      PDF infographic page added.

greggmiller committed:
All icons change to FontAwesome from Emojis in alignment with management request. Appearance improves profesisonalism.
Minor typos addressed.
Movement of your welfare section to Quick ICU training to reflect less significance.
Functional clinical support tool with high utility in patient management to help guide blood product administration. reduces information-dense algorithms into a simple calculator. Improves app functionality and use.
Inforgraphic page algorithms are derived from are also included in screen.

 GitHub issues #343, #344, #346, #348, #349, #351, #352 all addressed.

greggmiller committed:
All icons change to FontAwesome from Emojis in alignment with management request. Appearance improves profesisonalism.
Minor typos addressed.
Movement of your welfare section to Quick ICU training to reflect less significance.
Functional clinical support tool with high utility in patient management to help guide blood product administration. reduces information-dense algorithms into a simple calculator. Improves app functionality and use.
Inforgraphic page algorithms are derived from are also included in screen.

 GitHub issues #343, #344, #346, #348, #349, #351, #352 all addressed.

greggmiller committed:
All icons change to FontAwesome from Emojis in alignment with management request. Appearance improves profesisonalism.
Minor typos addressed.
Movement of your welfare section to Quick ICU training to reflect less significance.
Functional clinical support tool with high utility in patient management to help guide blood product administration. reduces information-dense algorithms into a simple calculator. Improves app functionality and use.
Inforgraphic page algorithms are derived from are also included in screen.

 GitHub issues #343, #344, #346, #348, #349, #351, #352 all addressed.

greggmiller committed:
All icons change to FontAwesome from Emojis in alignment with management request. Appearance improves profesisonalism.
Minor typos addressed.
Movement of your welfare section to Quick ICU training to reflect less significance.
Functional clinical support tool with high utility in patient management to help guide blood product administration. reduces information-dense algorithms into a simple calculator. Improves app functionality and use.
Inforgraphic page algorithms are derived from are also included in screen.

 GitHub issues #343, #344, #346, #348, #349, #351, #352 all addressed.

greggmiller committed:
All icons change to FontAwesome from Emojis in alignment with management request. Appearance improves profesisonalism.
Minor typos addressed.
Movement of your welfare section to Quick ICU training to reflect less significance.
Functional clinical support tool with high utility in patient management to help guide blood product administration. reduces information-dense algorithms into a simple calculator. Improves app functionality and use.
Inforgraphic page algorithms are derived from are also included in screen.

 GitHub issues #343, #344, #346, #348, #349, #351, #352 all addressed.

greggmiller committed:
All icons change to FontAwesome from Emojis in alignment with management request. Appearance improves profesisonalism.
Minor typos addressed.
Movement of your welfare section to Quick ICU training to reflect less significance.
Functional clinical support tool with high utility in patient management to help guide blood product administration. reduces information-dense algorithms into a simple calculator. Improves app functionality and use.
Inforgraphic page algorithms are derived from are also included in screen.

 GitHub issues #343, #344, #346, #348, #349, #351, #352 all addressed.

greggmiller committed:
All icons change to FontAwesome from Emojis in alignment with management request. Appearance improves profesisonalism.
Minor typos addressed.
Movement of your welfare section to Quick ICU training to reflect less significance.
Functional clinical support tool with high utility in patient management to help guide blood product administration. reduces information-dense algorithms into a simple calculator. Improves app functionality and use.
Inforgraphic page algorithms are derived from are also included in screen.

 GitHub issues #343, #344, #346, #348, #349, #351, #352 all addressed.

workerbee22 commented on ios/Podfile.lock:
```diff
@@ -231,4 +237,4 @@ SPEC CHECKSUMS:
 PODFILE CHECKSUM: 65d456bea7227b5dd6affeee023495211d15af47
 
-COCOAPODS: 1.9.1
+COCOAPODS: 1.9.3
```
Will have to retest and do CI changes to check new cocoapods version.

workerbee22 replied:
Little to no differences between versions. Changed to 1.9.1 to match CI setup.

workerbee22 commented on lib/rotem/advice_widget.dart:
```diff
@@ -0,0 +1,47 @@
+                itemBuilder: (context, index) {
+                  return Text(list[index].toString(),
+                      style: Styles.textSemiBold);
+                }),
```
Trailing commas is usually the way to go .... so },), which will then dartfmt correctly.

workerbee22 replied:
Fixed

workerbee22 committed:
Fix for Android back icon on appBar being white-on-white for Android 8.x.
Flutter format minor updates.
Cocoapods back to v1.9.1 to match CI setup.
Re tested on Android and iOS.

workerbee22 committed:
Minor tweaks for trailing commas.

workerbee22 committed:
Formatting fix.

workerbee22 reviewed (approved):
LGTM ... just fixed an Android specific issue and formatting.
